demos/c_api_minimal_app/README.md

```diff
@@ -0,0 +1,90 @@
+# C API inference demo (C/C++)
+
+This demo demonstrate how to use C API from the OpenVINO Model Server to create C and C++ application.
+Building the application is executed inside the docker container to illustrate end to end usage flow.
+
+## Build libovms_shared.so
+To build the capi docker image, you must first build the `ovms.tar.gz` package with the `libovms_shared.so` library and `ovms.h` header. 
+run `make` command in ovms git main directory.
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+make
+```
+
+## Prepare demo image
+Next enter the directory with the example and build the demo docker with all dependencies and examples that will be named `openvino/model_server-capi`.
+The example image also contains dummy model and config.json required for the applications.
+```bash
+cd demos/capi
```

Comment:
The directory has changed to c_api_minimal_app, change in all places accordingly

Reply:
Ok.